src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+  margin: 0;
+}
+
+.card__img,
+.card__product,
+.card__code,
+.card__rev,
+.card__rev-text,
+.card__price,
+.card__price-text,
+.card__cost,
+.card__button {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding: 33px 16px 16px;
+  width: 200px;
+
+  &__img {
+    margin-bottom: 40px;
+    height: 134px;
+  }
+
+  &__product {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-accent;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary;
+  }
+
+  &__rev {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $main-accent;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 16px;
+    color: $secondary;
+  }
+
+  &__cost {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-accent;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    border: 1px solid $blue-accent;
+    border-radius: 5px;
+    box-sizing: border-box;
+    background-color: $blue-accent;
+    color: #fff;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+  }
+
+  &:hover {
```

Comment:
Hover effects should not be applied to the entire card, but only to the clickable elements. Move hover effects on the button, not on the whole card.